File: src/net/BukkitPE/event/block/LeavesDecayEvent.java
package net.BukkitPE.event.block;

import net.BukkitPE.block.Block;
import net.BukkitPE.event.Cancellable;
import net.BukkitPE.event.HandlerList;

/**
 * author: MagicDroidX
 * BukkitPE Project
 */
public class LeavesDecayEvent extends BlockEvent implements Cancellable {

    private static final HandlerList handlers = new HandlerList();

    public static HandlerList getHandlers() {
        return handlers;
    }

    public LeavesDecayEvent(Block block) {
        super(block);
    }

}

File: src/BukkitPE/event/block/LeavesDecayEvent.php
<?php
namespace BukkitPE\event\block;

use BukkitPE\block\Block;
use BukkitPE\event\Cancellable;

class LeavesDecayEvent extends BlockEvent implements Cancellable{
	public static $handlerList = null;

	public function __construct(Block $block){
		parent::__construct($block);
	}

}